Ecal/src/Ecal/EcalClusterProducer.cxx

```diff
@@ -73,6 +72,35 @@ void EcalClusterProducer::produce(framework::Event& event) {
       cluster.addHits(wcVec[aWC].getHits());
       cluster.addFirstLayerHits(fWcVec[aWC].getHits());
 
+      float x(0), y(0), z(0), xx(0), yy(0), zz(0), n(0);
+      float w = 1;  // weight
```

Comment:
please try to avoid a single character variable names, as in I.1 of https://ldmx-software.github.io/developing/coding-rules.html